src/nyc_trees/js/src/map.js

```diff
@@ -22,6 +56,12 @@ function create(options) {
         zoomControl = L.control.zoom({position: 'bottomleft'}).addTo(map),
         $controlsContainer = $(zoomControl.getContainer());
 
+    map.setCenterAndZoomWM = function(zoom, location) {
+        setCenterAndZoomWM(map, zoom, location);
+    };
+
+    map.zoomLevels = Object.freeze(zoom);
```

Comment:
Instead of putting the zoom levels on the map object, consider adding it to the exports.  Also, it's probably a good idea to feature-detect before using `Object.freeze`:

```javascript
modules.exports = {
    create: create,
    zoom: Object.freeze ? Object.freeze(zoom) : zoom
}
```